projects/sentinel/layers/spark.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: Setup Spark master
+  hosts: localhost
+  connection: local
+  roles:
+    - role: aws
+      ec2_key_name: "eu-smartcat"
+      ec2_security_group: "sentinel"
+      ec2_subnet_id: "subnet-9c0af4e6"
+      ec2_image_id: ami-26c43149
+      ec2_instance_type: t2.micro
+      ec2_region: eu-central-1
+      ec2_count: 1
+      ec2_tags:
+        Name: "sentinel-spark-master"
+      ec2_group: sentinel-spark-master
+
+- name: Install software
+  hosts: sentinel-spark-master
+  gather_facts: False
+  remote_user: ubuntu
+  become: yes
+  become_method: sudo
+  tags:
+    - aws-setup
+    - aws-start
+  pre_tasks:
+    - name: Gathering facts
+      setup:
+      tags:
+        - spark
+  roles:
+    - { role: spark, tags: ["spark"] }
+  post_tasks:
+    - name: Start master
+      shell: "{{ spark_home }}/sbin/start-master.sh"
+
+- name: Setup Spark slaves
+  hosts: localhost
+  connection: local
+  roles:
+    - role: aws
+      ec2_key_name: "eu-smartcat"
+      ec2_security_group: "sentinel"
+      ec2_subnet_id: "subnet-9c0af4e6"
+      ec2_image_id: ami-26c43149
+      ec2_instance_type: t2.medium
+      ec2_region: eu-central-1
+      ec2_count: 3
+      ec2_tags:
+        Name: "sentinel-spark-slave"
+      ec2_group: sentinel-spark-slaves
```

Comment:
How does this work, is it separate Spark cluster or our Spark slaves are collocated with Cassandra nodes to work locally with data?